pycytominer/consensus.py

```diff
@@ -109,7 +109,7 @@ def consensus(
             population_df=profiles,
             replicate_columns=replicate_columns,
             features=features,
-            **modz_args
+            **modz_args,
```

Comment:
Double checking: was this a Black formatting change of some kind?

Reply:
@d33bs Somehow some non-black code was in our codebase. Not sure if it get missed during the Great Blackening, or if it got committed afterwards. Either way, one of the pipeline changes for this build is that pre-commit checks are run on the entire codebase rather than just changed files. That should prevent this from happening again in the future.